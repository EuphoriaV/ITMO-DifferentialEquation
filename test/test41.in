5.35
((((8.67 * (1.77 * x)) + (7.62 + x)) * (x + x)) * ((6.91 - 4.99) * (((x * x) + 4.17) + (((10.0 * 1.97) * x) * (7.61 + ((6.51 + x) + (x * 7.95)))))))
5.15
5.8