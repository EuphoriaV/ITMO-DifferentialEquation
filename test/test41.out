2.5E-6